.github/workflows/build.yml

```diff
@@ -12,12 +12,37 @@ env:
   AWS_NO_SIGN_REQUEST: "1"
 
 jobs:
+  # Increase rate limits for public registry
+  amazon-ecr-auth:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: 'arn:aws:iam::276514628126:role/ci-kart'
+          aws-region: us-east-1
+          mask-aws-account-id: 'false'
+          role-session-name: kart-ci-gha
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v2
+        with:
+          registry-type: public
+          mask-password: 'false'
```

Comment:
this doesn't seem necessary / a good idea? even if we're not using debug-level logging atm (are we?)

Reply:
It's how they document it: https://github.com/aws-actions/amazon-ecr-login?tab=readme-ov-file#run-an-image-as-a-service